solanum-tuberosums/djangazon
# Added date_added field to Product Model

## Associated Ticket
[User can view latest products](https://github.com/solanum-tuberosums/djangazon/issues/18)
[User can view product details](https://github.com/solanum-tuberosums/djangazon/issues/13)

## Description 
Added the ```date_added``` field to the Product model.

## To Test
- Make sure you delete the **contents** of your migrations folder (**NOT** the ```__init__.py``` file)
- _Possibly Optional_: delete db.sqlite3
- Then run ```python manage.py makemigrations website```
- Then run ```python manage.py migrate```
- Then make a new product category
- Then make a new product
- Then view products to ensure that the product you added has a date value that matches the date you added it

## All Tests Pass
N/A
